composites/Plugin/SnippetEditor/components/ReplacementVariableEditor.js

```diff
@@ -99,6 +99,7 @@ class ReplacementVariableEditor extends React.Component {
 					id={ fieldId }
 				>
 					<ReplacementVariableEditorStandalone
+						fieldId={ `${ fieldId }-field` }
```

Comment:
Suffixing a field with `-field` seems a little strange in terms of naming, especially when you look at the snapshots, which contain fields that have the word 'field' in there twice. Example: `snippet-editor-field-title-field`.

Reply:
We can't rename the earlier id we introduced because of backwards compatibility.